File: qr_create.js
const fs = require('fs');
const QRCode = require('qrcode-svg');

// URLs to generate QR codes for
const urls =  [
    "example_url"
];

// Loop through the URLs and generate QR codes
for (let i = 0; i < urls.length; i++) {
    const qrCode = new QRCode(urls[i]);
    const svgString = qrCode.svg();

    const fileName = `input/qrcode_${i + 1}.svg`;
    fs.writeFileSync(fileName, svgString);

    console.log(`QR code ${i + 1} saved as ${fileName}`);
}

File: qr_create.py
import qrcode
import qrcode.image.svg

# Your input data (e.g., a URL)
data = "Here is your present Samantha. -Q 12/1/23"

# Create a QR code instance
qr = qrcode.QRCode(
    version=1,  # Adjust version as needed
    error_correction=qrcode.constants.ERROR_CORRECT_L,
    box_size=10,
    border=4,
)

# Add the data to the QR code
qr.add_data(data)
qr.make(fit=True)

# Create an SVG image factory
factory = qrcode.image.svg.SvgPathImage

# Generate the SVG QR code
svg_image = qr.make_image(image_factory=factory)

# Save the SVG QR code to a file
with open("input/qrcode.svg", "wb") as f:
    svg_image.save(f)
